build.gradle.kts

```diff
@@ -38,7 +37,6 @@ configurations.implementation {
 }
 
 dependencies {
-  shadow(gradleApi())
```

Comment:
declaring a shadow dependency on the local gradle api fails to publish as we can't bundle this. It also shouldn't be needed